### ISSUE TITLE
Spec property decorations fail accessibility guidelines

### ISSUE THREAD

**Issue by ejratl** [oasis-tcs/cti-stix2#310]
[WCAG Level AAA](https://www.w3.org/WAI/WCAG2AAA-Conformance) requires a 7:1 contrast for normal text (<14 pt font). The spec font is 10 points and the decorators have the following colors:
Property fg: #C7254E bg: #F9F2F4  
Literal fg: #073763 bg: #CFE2F3 
Examples fg: #000000 bg: #EFEFEF 

The literal decorators pass with a score of 9.1:1. The examples decorators pass with a score of 18.26:1. However the property decorators fail with a contrast of 5:1 per the [WebAccessibility Contrast Checker](https://webaim.org/resources/contrastchecker/).

It would be nice to fix this accessibility issue in a future revision.

**Comment by srrelitz2** [comment]
Agree that change needs to be applied.  Actual color change is still TBD
